Source/PINOperationGroup.m

```diff
@@ -167,6 +158,20 @@ - (void)waitUntilComplete
 {
   [self start];
   dispatch_group_wait(_group, DISPATCH_TIME_FOREVER);
+  [self runCompletionIfNeeded];
```

Comment:
Hmm but right now the completion queue is privately hard-coded to the default-priority queue. The user doesn't even know about the completion queue, so the documentation wouldn't make sense.